src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 15px 14px;
+  height: 403px;
+}
+
+.product-card__img {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 15px;
+}
+
+.product-card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  padding-top: 45px;
+  margin-bottom: 5px;
+}
+
+.product-card__code {
+  margin-bottom: 15px;
+}
+
+.product-card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__img {
+  background: url("images/star.svg") no-repeat center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.stars__img:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__img,
+.stars--1 .stars__img:first-child,
+.stars--2 .stars__img:nth-child(-n+2),
+.stars--3 .stars__img:nth-child(-n+3),
+.stars--4 .stars__img:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.product-card__money {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.product-card__buy {
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  border: 1px solid transparent;
+
+}
+
+.product-card:hover .product-card__buy {
+  background-color: #fff;
+  color: #00acdc;
+  border-color: #00acdc;
+  transition: all 0.2s linear;
```

Comment:
Don't set transition all. Indicate what rules you want to use transition for. And move transition to .product-card__buy selector.